### ISSUE TITLE
Correct code errors in fw_chunks(self)

### ISSUE THREAD

**Issue by Irelandr** [salesforce/fsnet#9]
idx = idx.unsqueeze(1).float()
            old_w = ww @ idx
→
v = v.unsqueeze(1).float()
            old_w = ww @ v

**Comment by Irelandr** [comment]
may be you need v.detach()

**Comment by LiuYasuo** [comment]
I have found the same problem in this code. Moreover, did you notice the memory leak issue when running this code.

**Comment by Irelandr** [comment]
No，I ran fsnet on ETTh and there were no problems.

**Comment by LiuYasuo** [comment]
If you have the time, I hope you could run the original code of FSnet again. And you can use the "top" command to check the memory usage of this process while it is running. Then you will find that the memory occupied by this process continues to increase.

**Comment by Irelandr** [comment]
It seems that I didn't encounter the problem you said.

**Comment by LiuYasuo** [comment]
Could you give me your way of contact such as Wechat,? I hope we could discuss this problem in more detail. Thanks!

**Comment by Irelandr** [comment]
it's irelandr

**Comment by ooooooover** [comment]
> Could you give me your way of contact such as Wechat,? I hope we could discuss this problem in more detail. Thanks!

Can we add a wechat to chat in detail, I have some doubts too.Wechat can be sent to my email: zhangweiyang12138@gmail.com
